src/test/java/com/moorhuhnservice/moorhuhnservice/MoorhuhnServiceApplicationTests.java

```diff
@@ -0,0 +1,153 @@
+package com.moorhuhnservice.moorhuhnservice;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.moorhuhnservice.moorhuhnservice.data.Configuration;
+import com.moorhuhnservice.moorhuhnservice.data.ConfigurationDTO;
+import com.moorhuhnservice.moorhuhnservice.data.Question;
+import com.moorhuhnservice.moorhuhnservice.data.QuestionDTO;
+import com.moorhuhnservice.moorhuhnservice.data.mapper.ConfigurationMapper;
+import com.moorhuhnservice.moorhuhnservice.data.mapper.QuestionMapper;
+import com.moorhuhnservice.moorhuhnservice.repositories.ConfigurationRepository;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import javax.servlet.http.Cookie;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+@AutoConfigureMockMvc
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class MoorhuhnServiceApplicationTests {
+
+  @Autowired
+  private MockMvc mvc;
+
+  @Autowired
+  private ConfigurationMapper configurationMapper;
+
+  @Autowired
+  private QuestionMapper questionMapper;
+
+  @Autowired
+  private ConfigurationRepository configurationRepository;
+
+  private final String API_URL = "/api/v1/minigames/moorhuhn/configurations";
+  private ObjectMapper objectMapper;
+  private Configuration createdConfiguration;
+
+  @BeforeEach
+  public void createBasicData() {
+    configurationRepository.deleteAll();
+    Question questionOne = new Question("Are you cool?", "Yes", Arrays.asList("No", "Maybe"));
```

Comment:
NoArgsConstructor und setter sind einfacher zu lesen

Reply:
done in newest commit